src/common/xo/create-network-modal/index.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react'
+import { injectIntl } from 'react-intl'
+
+import SingleLineRow from '../../single-line-row'
+import _, { messages } from '../../messages'
+import { SelectPif } from '../../select-objects'
+import { Col } from '../../grid'
+import { createSelector } from 'selectors'
```

Comment:
Group imports: on one side the global imports, on the other side local imports (with relative paths).